.github/ISSUE_TEMPLATE/release-template.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Release Process
+about: COMMITTER ONLY: Managing the Jetty release process
+title: 'Jetty Releases 9.4.x, 10.0.y, 11.0.y'
+assignees: ''
+labels: Build
+
+---
+
+**Jetty Versions:**
+This release process will produce releases:
+
+**Target Date:**
+
+**Tasks:**
+- [x] Create the release(s) issue.
+- [ ] Update the target Jetty version(s) in the issue.  
+- [ ] Update the target release date in the issue.
+- [ ] Link this issue to the target [project(s)](https://github.com/eclipse/jetty.project/projects).
+- [ ] Assign this issue to a "release manager".
+- [ ] Review [draft security advisories](https://github.com/eclipse/jetty.project/security/advisories). Ensure that issues are created and assigned to GitHub Projects to capture any advisories that will be announced.
+- [ ] Review their issues/PRs assigned to the target [project(s)](https://github.com/eclipse/jetty.project/projects).  Any PRs that are moved to subsequent releases should be commented on so their authors are informed.
+- [ ] Freeze the target [project(s)](https://github.com/eclipse/jetty.project/projects) by editing their names to "Jetty X.Y.Z FROZEN"
+- [ ] Wait 24 hours from last change to the issues/PRs included in a FROZEN GitHub Project.
+- [ ] Verify target [project(s)](https://github.com/eclipse/jetty.project/projects) are complete.
+- [ ] Verify that branch `jetty-10.0.x` is merged to branch `jetty-11.0.x`.
+- [ ] Assign issue to "build manager", who will stage the releases.
+   + [ ] Ensure `VERSION.txt` additions for each release will be meaningful, descriptive, correct text.
+   + [ ] Stage Release(s).
+   + [ ] Edit a draft release (for each Jetty release) in GitHub (https://github.com/eclipse/jetty.project/releases). Content is generated with the "changelog tool".
+- [ ] Assign issue to "test manager", who will oversee the testing of the staged releases.
+   + [ ] Add [testing tick list](https://github.com/eclipse/jetty.project/blob/jetty-10.0.x/.github/test-ticklist.md) to the target [project(s)](https://github.com/eclipse/jetty.project/projects).
```

Comment:
The testing tick list is the same for all Jetty versions. I fail to see why needs to be split into different projects.
It's the same as "Stage releases" which is here, not split per-project.
IMHO there should be just 1 tick list to keep things simpler -- one place to look only.

Reply:
I'm on the fence with this one.   I can see @sbordet's point that it makes us have to look in multiple places for status.  I can also see that if we had to reroll just one release it would be easier to track iterations.   But then how likely is a single release re-roll?  Typically we will need to reroll all the releases so a single testing summary checklist would work.   Others need to comment else I think we will revert to a single list.